src/Plugins/StateService/Verification/VerificationContext.cs

```diff
@@ -27,7 +27,7 @@ namespace Neo.Plugins.StateService.Verification
 {
     class VerificationContext
     {
-        private const uint MaxValidUntilBlockIncrement = 100;
+        private const uint MaxValidUntilBlockIncrement = 100; // Change to 500!??
```

Comment:
Whoa. Unexpected one. It's totally different from the "regular" MVUBI, but 500 here would let more ExtensiblePayloads with signatures float around the network, I doubt the intention here was to have it related to time.

Reply:
We should related it with the timeblock

Reply:
I think you guys should decide, that is why I left as comment.